x-pack/plugins/fleet/server/services/agents/update_agent_tags.ts

```diff
@@ -44,30 +42,17 @@ export async function updateAgentTags(
       }
     }
   } else if ('kuery' in options) {
-    const batchSize = options.batchSize ?? SO_SEARCH_LIMIT;
-    const res = await getAgentsByKuery(esClient, {
-      kuery: options.kuery,
-      showInactive: options.showInactive ?? false,
-      page: 1,
-      perPage: batchSize,
-    });
-    if (res.total <= batchSize) {
-      givenAgents = res.agents;
-    } else {
-      return await new UpdateAgentTagsActionRunner(
-        esClient,
-        soClient,
-        {
-          ...options,
-          batchSize,
-          total: res.total,
-          kuery: options.kuery,
-          tagsToAdd,
-          tagsToRemove,
-        },
-        { pitId: '' }
-      ).runActionAsyncWithRetry();
-    }
+    return await new UpdateAgentTagsActionRunner(
```

Comment:
simplified the logic to use retry for all update tags kuery actions
as reported [here](https://github.com/elastic/kibana/issues/144161#issuecomment-1353185926), the version conflict happened even with less than 10k agents, which didn't retry before

Could reproduce in an ECE instance by adding a tag on 5k horde agents and getting this response from bulk API:
```
{"statusCode":500,"error":"Internal Server Error","message":"version conflict of 1865 agents"}
```

Reply:
tested on pr cloud deployment that 5000 agents update tags is done successfully (async)
<img width="2664" alt="image" src="https://user-images.githubusercontent.com/90178898/208427581-3e2fca87-266f-4ab2-a2e3-2b43fd129d4f.png">